### DISCUSSION TITLE
Can sonar.oe.databases be setup as a relative path?

### DISCUSSION THREAD

**Discussion by JayMartin0718** [Riverside-Software/sonar-openedge#960] [Q&A]
sonar.oe.propath = c:\git\dbs
sonar.oe.databases = /cust.df, /orders.df

My .df files are in c:\git\dbs, however it appears I must enter an absolute path for databases (c:\git\dbs\cust.df).

**Comment by gquerret** [comment]
The paths are relative to the work directory (ie where the sonar-scanner process is started).

**Reply by JayMartin0718** [reply]
I mean relative to sonar.oe.propath.

How do I determine where the sonar-scanner process is started? In my case, it starts as a plug-in.

**Reply by gquerret** [reply]
> I mean relative to sonar.oe.propath.

No, it's not relative to propath, but to the work directory.

> How do I determine where the sonar-scanner process is started? In my case, it starts as a plug-in.

Depends on the plugin... You can always try to execute `echo $pwd` or the windows equivalent in your process.
